packages/service-library/requirements/_aiohttp.in

```diff
@@ -7,11 +7,11 @@
 
 openapi-core==0.12.0     # frozen until https://github.com/ITISFoundation/osparc-simcore/pull/1396 is CLOSED
 lazy-object-proxy~=1.4.3 # cannot upgrade due to contraints in openapi-core
-aiozipkin~=0.7.1         # 1.X has an issue with no content responses
-
+aiozipkin
 
 aiohttp
 aiopg[sa]
+nest_asyncio # used for aiozipkin initialization (we should move to OpenTelemetry)
```

Comment:
could you please open an issue with what you envision using open-telemetry, e.g. basic ideas, how it gets integrated and steps to achieve it?  Add then here a reference to the issue here

Reply:
done